src/views/Auth/Login.test.jsx

```diff
@@ -39,17 +34,17 @@ it('should allow the user to log in', async () => {
 
   expect(container).toMatchSnapshot();
 
-  const emailField = screen.getByLabelText('Email');
-  const passwordField = screen.getByLabelText('Password');
+  const emailField = screen.getByLabelText(/email/i);
```

Comment:
These two formats are actually equivalent to each other. This simply changes the lookup to use RegEx instead of the literal text within the Label element.